addon/ng2/blueprints/ng2/files/package.json

```diff
@@ -42,7 +42,9 @@
     "jasmine-spec-reporter": "^2.4.0",
     "karma": "^0.13.15",
     "karma-chrome-launcher": "^0.2.3",
+    "karma-coverage": "^1.0.0",
     "karma-jasmine": "^0.3.8",
+    "karma-remap-istanbul": "git+https://github.com/marines/karma-remap-istanbul.git#patch-1",
```

Comment:
I agree. So, there are two options:

1. We wait for the dev to implement the PR for `karma-remap-istanbul`
2. We/I/Someone creates `karma-remap-istanbul2` with the fix.

Suggestions?